client/api_object.go

```diff
@@ -886,6 +886,10 @@ func (c *client) ListObjects(ctx context.Context, bucketName string, opts types.
 			return types.ListObjectsResult{}, fmt.Errorf("continuation-token does not match the input prefix")
 		}
 	}
+	includeRemoved := "false"
+	if opts.ShowRemovedObject {
```

Comment:
it is strange to set a string called "false" ,  can we  change it  to this logic ?  if includeRemoved == false , you can just not set the the "include-removed" params, and if includeRemoved == true , you can set the params   "include-removed" with empty value of ""

Reply:
fixed